index.js

```diff
@@ -69,23 +75,176 @@ module.exports = function (homebridge) {
         .then((json) => {
           if (utils.isMessageOfTypeValues(json)) {
             // bulk update ("request values")
-            this.deviceStates = {};
-            for (let item of json) {
-              for (let device of item.devices) {
-                this.deviceStates[device] = item.values.state === 'on';
-              }
-              this.log(`${this.id}: Updated internal states of devices: ${item.devices}`);
+            var item = json.find((item) => {
+              return item.devices.indexOf(this.config.deviceId) !== -1;
+            });
+            if (item) {
+              this.deviceState = item.values.state === 'on';
+              this.log(`Initialized device with state "${item.values.state}"`);
+            } else {
+              this.log(`Could not find device with id "${this.config.deviceId}"`);
             }
           } else if (utils.isMessageOfTypeUpdate(json)) {
-            // item update (after "control")
-            for (let device of json.devices) {
-              this.deviceStates[device] = json.values.state === 'on';
+            // item update (after "control")'use strict';
+            const WebSocketClient = require('websocket').client;
+            const utils = require('./lib/utils');
+
+            const pluginName = 'homebridge-pilight';
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).